### PR TITLE
LRAC-7203 JS Client requests to Asah should contain projectId

### PR DESCRIPTION
**Pull request by fcodias**
https://issues.liferay.com/browse/LRAC-7203

### CODE REVIEW

**Comment by fcodias** [comment]
ci:test:sf

File: modules/apps/analytics/analytics-client-js/src/utils/constants.js [hunk 1]
```diff
@@ -16,6 +16,10 @@ export const DEBOUNCE = 1500;
 
 export const FLUSH_INTERVAL = 2000;
 
+//Custom Headers
```
Comment (andreldm) [inline modules/apps/analytics/analytics-client-js/src/utils/constants.js]: SF

**Comment by andreldm** [comment]
LGTM

**Comment by fcodias** [comment]
ci:forward

**Pull request closed without merging**